bears/c_languages/CPPCheckBear.py

```diff
@@ -30,18 +33,37 @@ class CPPCheckBear:
 
     def create_arguments(self, config_file,
                          enable: typed_list(str) = [],
+                         language: str = '',
```

Comment:
Use `language: Language = Language['Unknown']`. This will set a default of `None`.

Reply:
Thanks @rajdeepbharati for your review!

Reply:
So, if I input `--language=c++`, it will be treated as `"c++"`.
But what I need to do is that it should be like `Language['c++']`.

Reply:
Hello @gutsytechster do you still plan to work on this ? We need it for our game  (play0ad.com)  

You can't do what you want in one line. This won't work :
```python
def create_arguments(self, config_file, enable: typed_list(str) = [], language: str = Language[language]):
```
From what I understand the default should be language = None
```python
def create_arguments(self, config_file, enable: typed_list(str) = [], language: str = None):
```
Then line 55 check for  
```python
        if language is not None:
            try:
                lang = Language[language]
                if isinstance(lang, Language.CPP):
                    args += ('--language=c++',)
                elif isinstance(lang, Language.C):
                    args += ('--language=c',)
                else:
                    logging.error('Language can be either c or c++')
                    return
            except(AttributeError):
                logging.error('Language can be either c or c++')
                return
        else
            args += ('--language=c',)
```

Reply:
@StanleySweet, thanks for update on this.
Yes, I'll finish the issue :)

Reply:
Cool. I think you need to add tests now. The Appveyor fails because the coberage went from 100% to 87 something 
Not sure why travis fails but I gurss it's related
Do younow how to write the tests ?

Reply:
Oh yeah! I am trying to resolve these issues